src/amo/components/Collection/index.js

```diff
@@ -0,0 +1,202 @@
+/* @flow */
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { withErrorHandler } from 'core/errorHandler';
+import log from 'core/logger';
+import translate from 'core/i18n/translate';
+import { parsePage } from 'core/utils';
+import { fetchCollection, fetchCollectionPage } from 'amo/reducers/collections';
+import type {
+  CollectionsState,
+  CollectionType,
+} from 'amo/reducers/collections';
+import AddonsCard from 'amo/components/AddonsCard';
+import Link from 'amo/components/Link';
+import NotFound from 'amo/components/ErrorPage/NotFound';
+import Paginate from 'core/components/Paginate';
+import Card from 'ui/components/Card';
+import LoadingText from 'ui/components/LoadingText';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { ReactRouterLocation } from 'core/types/router';
+
+import './styles.scss';
+
+
+type Props = {|
```

Comment:
I thought we usually called these "PropTypes". I don't care which ones but we should be consistent.

Reply:
Actually I think I slightly prefer "PropTypes" 😉 because I think it helps not confuse things with actual props and signifies intent better to users used to using `prop-types`

Reply:
Flow uses `Props` https://flow.org/en/docs/react/components/, that's why I use `Props`, but I am fine with `PropTypes`.